cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -713,3 +716,67 @@ def test_custom_button_open_url_service_obj(objects, button_group):
         1550002
     """
     pass
+
+
+@pytest.fixture(params=["Service", "Other"], scope="module")
+def unassigned_btn_setup(request, appliance, provider, service):
+    if request.param == "Service":
+        btn_type = request.param
+        obj = MyService(appliance, name=service.name)
+        destinations = [ViaUI, ViaSSUI]
+    else:
+        btn_type = "Provider"
+        obj = provider
+        destinations = [ViaUI]
+
+    gp = appliance.collections.button_groups.instantiate(
+        text="[Unassigned Buttons]", hover="Unassigned buttons", type=btn_type
+    )
+    yield obj, gp, destinations
+
+
+@pytest.mark.provider([VMwareProvider], override=True, scope="module", selector=ONE)
+def test_custom_button_unassigned_behavior_objs(
+    appliance, setup_provider, unassigned_btn_setup, request
+):
+    """ Test unassigned custom button behavior
+
+    Note: Service unassigned custom button should display on SSUI but not OPS UI.
+    For other than service objects also fallows same behaviour l.e. not display on OPS UI.
```

Comment:
Typo : 
Note: Service unassigned custom button should display on SSUI but not OPS UI.
    For other than service objects also **follows** same behaviour **i.e.** not display on OPS UI.